Cannot build package gcc for target i686-w64-mingw32.static

Attempting to compile mxe using the source code obtained with `git clone https://github.com/mxe/mxe.git` fails to build gcc with the message below. I tried with the gcc version 5.4.0 that is current on Xubuntu 16.04LTS (32 bit), and also with gcc version 7.2.0, since in #1361 it says 'fixed upstream with gcc'.

```
ihp-pc49>make gcc
[build nonetwork lib]
[check requirements]
[git-log]   0a829f9 Update packages.json & build-matrix.html
[build]     cmake-conf             x86_64-pc-linux-gnu
[build]     mxe-conf               x86_64-pc-linux-gnu
[done]      cmake-conf             x86_64-pc-linux-gnu                                      12 KiB         0m3.627s
[done]      mxe-conf               x86_64-pc-linux-gnu                                      328 KiB        0m5.662s
[build]     cmake                  x86_64-pc-linux-gnu
[build]     geoip-database         x86_64-pc-linux-gnu
[build]     gmp                    x86_64-pc-linux-gnu
[no-build]  googlemock             x86_64-pc-linux-gnu
[no-build]  googletest             x86_64-pc-linux-gnu
[no-build]  lzma                   x86_64-pc-linux-gnu
[no-build]  libtool                x86_64-pc-linux-gnu
[build]     ragel                  x86_64-pc-linux-gnu
[build]     ucl                    x86_64-pc-linux-gnu
[build]     zlib                   x86_64-pc-linux-gnu
[build]     waf                    x86_64-pc-linux-gnu
[build]     yasm                   x86_64-pc-linux-gnu
[done]      geoip-database         x86_64-pc-linux-gnu                                      6444 KiB       0m6.886s
[build]     pkgconf                x86_64-pc-linux-gnu
[done]      waf                    x86_64-pc-linux-gnu                                      8628 KiB       0m8.403s
[build]     protobuf               x86_64-pc-linux-gnu
[done]      zlib                   x86_64-pc-linux-gnu                                      3936 KiB       0m10.633s
[done]      ragel                  x86_64-pc-linux-gnu                                      14568 KiB      0m18.755s
[done]      pkgconf                x86_64-pc-linux-gnu                                      1520 KiB       0m12.293s
[done]      yasm                   x86_64-pc-linux-gnu                                      33572 KiB      0m19.174s
[done]      ucl                    x86_64-pc-linux-gnu                                      6068 KiB       0m29.698s
[build]     upx                    x86_64-pc-linux-gnu
[done]      upx                    x86_64-pc-linux-gnu                                      25908 KiB      0m14.535s
[done]      gmp                    x86_64-pc-linux-gnu                                      29708 KiB      0m54.750s
[build]     isl                    x86_64-pc-linux-gnu
[build]     mpfr                   x86_64-pc-linux-gnu
[done]      isl                    x86_64-pc-linux-gnu                                      42332 KiB      0m31.691s
[done]      mpfr                   x86_64-pc-linux-gnu                                      15544 KiB      0m37.975s
[build]     mpc                    x86_64-pc-linux-gnu
[done]      mpc                    x86_64-pc-linux-gnu                                      5608 KiB       0m17.109s
[done]      protobuf               x86_64-pc-linux-gnu                                      459196 KiB     2m35.954s
[done]      cmake                  x86_64-pc-linux-gnu                                      356012 KiB     5m30.047s
[build]     cmake-conf             i686-w64-mingw32.static
[build]     mxe-conf               i686-w64-mingw32.static
[done]      mxe-conf               i686-w64-mingw32.static                                  12 KiB         0m3.639s
[done]      cmake-conf             i686-w64-mingw32.static                                  12 KiB         0m3.654s
[build]     pkgconf                i686-w64-mingw32.static
[no-build]  mingw-w64              i686-w64-mingw32.static
[done]      pkgconf                i686-w64-mingw32.static                                  424 KiB        0m3.604s
[build]     binutils               i686-w64-mingw32.static
[done]      binutils               i686-w64-mingw32.static                                  477772 KiB     0m46.163s
[build]     gcc                    i686-w64-mingw32.static

Failed to build package gcc for target i686-w64-mingw32.static!
------------------------------------------------------------
Makefile:4118: recipe for target 'all-gcc' failed
make[2]: *** [all-gcc] Error 2
make[2]: Leaving directory '/LocalHD/mxe/tmp-gcc-i686-w64-mingw32.static/gcc-5.4.0.build_'
Makefile:655: recipe for target 'build-only-gcc_i686-w64-mingw32.static' failed
make[1]: *** [build-only-gcc_i686-w64-mingw32.static] Error 2
make[1]: Leaving directory '/LocalHD/mxe'
real	0m55.092s
user	1m35.220s
sys	0m9.364s
------------------------------------------------------------
[log]      /LocalHD/mxe/log/gcc_i686-w64-mingw32.static

Makefile:655: recipe for target '/LocalHD/mxe/usr/i686-w64-mingw32.static/installed/gcc' failed
make: *** [/LocalHD/mxe/usr/i686-w64-mingw32.static/installed/gcc] Error 1
ihp-pc49>gcc --version
gcc (GCC) 7.2.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

```

--- a/Makefile
+++ b/Makefile
@@ -52,7 +52,7 @@ TIMESTAMP  := $(shell date +%Y%m%d_%H%M%S)
 PKG_DIR    := $(PWD)/pkg
 TMP_DIR     = $(MXE_TMP)/tmp-$(1)
 BUILD      := $(shell '$(EXT_DIR)/config.guess')
-PATH       := $(PREFIX)/$(BUILD)/bin:$(PREFIX)/bin:$(PATH)
+PATH       := $(PREFIX)/$(BUILD)/bin:$(PREFIX)/bin:$(shell echo $$PATH | $(SED) -e 's,:\.$$,,' -e 's,\.:,,g')
 
 # set to empty or $(false) to disable stripping
 STRIP_TOOLCHAIN := $(true)

